WhaleSpotting/react-app/src/styles/Profile.scss

```
@@ -0,0 +1,116 @@
+@import './Constants.scss';
+
+.body {
+    max-width: 1024px;
+    font-family: $fontFamily;
+    display: flex;
+    margin-bottom: $footerHeight;
+    margin-top: $headerHeight;
```

You still need the auto left and right, so I would suggest
```suggestion
    margin: $headerHeight auto $footerHeight auto;
```